src/components/ThemeMenu/Modal/ThemeModal.tsx

```diff
@@ -0,0 +1,246 @@
+                            aria-labelledby="demo-radio-buttons-group-label"
+                            name="radio-buttons-group"
+                        >
+                            <FormControlLabel value={'HaskiTheme'} control={<Radio />} label={<Box display="flex" alignItems="center">
+                                <Typography>{t('components.ThemeModal.standardTheme')}</Typography>
+                                <LightMode sx={{ mr: 1 }} />
+                            </Box>} />
+                            <FormControlLabel value={'DarkTheme'} control={<Radio />} label={<Box display="flex" alignItems="center">
+                                <Typography>{t('components.ThemeModal.darkTheme')}</Typography>
+                                <DarkMode sx={{ mr: 1 }} />
+                            </Box>} />
+                            <FormControlLabel value={'AltTheme'} control={<Radio />} label={<Box display="flex" alignItems="center">
+                                <Typography>{t('components.ThemeModal.altTheme')}</Typography>
+                                <Brush sx={{ mr: 1 }} />
+                            </Box>} />
+                        </RadioGroup>
+                    </FormControl>
+                </Box>
+
+                <Divider
+                    orientation={isSmallScreen ? 'horizontal' : 'vertical'}
+                    flexItem
+                    sx={{
+                    width: isSmallScreen ? '100%' : 'auto',
+                    height: isSmallScreen ? 'auto' : '100%',
+                }}/>
+
+                {/**Right Column / Lower Row - Provides themed preview of radio selection and operating buttons*/}
+                <Box sx={rightColumnStyleComponent}>
+                    <Fab
+                        id="switch-page-right-theme-button"
+                        color="primary"
+                        data-testid={'ThemeModal-Right-Button'}
+                        onClick={() => changePage(1)}
+                        style={{
+                            position: 'sticky',
+                            top: '45%',
+                            left: '93.5%'
+                        }}>
+                        <ArrowForward />
+                    </Fab>
+                    <Fab
+                        id="switch-page-left-theme-button"
+                        color="primary"
+                        data-testid={'ThemeModal-Left-Button'}
+                        onClick={() => changePage(-1)}
+                        style={{
+                            position: 'sticky',
+                            top: '45%',
+                            left: '1.5%'
+                        }}>
+                        <ArrowBack />
+                    </Fab>
+                    <Fab
+                        id="close-theme-button"
+                        color="primary"
```

Comment:
Farbe vom Theme nehmen, wie bereits in anderen Componenten. 
(color: (theme) => theme.palette.primary.main)